sdk/src/main/java/io/opentelemetry/sdk/correlationcontext/CorrelationContextSdk.java

```diff
@@ -127,6 +129,18 @@ public int hashCode() {
       return this;
     }
 
+    @Override
+    public CorrelationContext.Builder setParent(Context context) {
+      Utils.checkNotNull(context, "context");
```

Comment:
Should `null` be allowed here (and in other setParent methods).

Reply:
Oh, that's a good call. Yes, it should.

Reply:
Actually I just realized other setParent methods now raise an error upon receiving `null` values.